misc/python/materialize/output_consistency/data_type/data_type_category.py

```diff
@@ -23,3 +23,5 @@ class DataTypeCategory(Enum):
     NUMERIC = 101
     BOOLEAN = 102
     TEXT = 103
+
+    ENUM = 200
```

Comment:
Do these values somehow correspond to Mz internal numbers or do you just choose them randomly?

Reply:
Just randomly